src/App.jsx

```diff
@@ -2,7 +2,7 @@ import * as React from 'react';
 import TodoList from './TodoList';
 import AddTodoForm from './AddTodoForm';
 
-const useSmiPersistantState = () => {
+const useSemiPersistentState = () => {
```

Comment:
You were supposed to move the `useEffect` and `useState` hooks back inside the `App` component and delete this function.